patches/server/0979-ignoring-moved-too-quickly-message-for-teleports.patch

```diff
@@ -0,0 +1,24 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: NonSwag <mrminecraft00@gmail.com>
+Date: Thu, 11 May 2023 17:05:44 +0200
+Subject: [PATCH] ignoring moved too quickly message for teleports
+
+
+diff --git a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+index 04a92f33f15d1696e38d38839651adf7d0462cac..2a905674ef59f1de7d1aeada2c10a1ebf769daab 100644
+--- a/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
++++ b/src/main/java/net/minecraft/server/network/ServerGamePacketListenerImpl.java
+@@ -1471,9 +1471,10 @@ public class ServerGamePacketListenerImpl implements ServerPlayerConnection, Tic
+                                 this.internalTeleport(this.player.getX(), this.player.getY(), this.player.getZ(), this.player.getYRot(), this.player.getXRot(), Collections.emptySet());
+                                 return;
+                             }
+-                            // Paper end
+-
+-                            if (!this.player.isChangingDimension() && (!this.player.getLevel().getGameRules().getBoolean(GameRules.RULE_DISABLE_ELYTRA_MOVEMENT_CHECK) || !this.player.isFallFlying())) {
++                            // Ignoring teleports as they unnecessarily flag the player
++                            // if (!this.player.isChangingDimension() && (!this.player.getLevel().getGameRules().getBoolean(GameRules.RULE_DISABLE_ELYTRA_MOVEMENT_CHECK) || !this.player.isFallFlying())) {
```

Comment:
why did you comment it out?

Reply:
So it is visible what was changed but if you want I can remove it